torch/_inductor/overrides.py

```diff
@@ -40,6 +40,11 @@ def replace_fx(gm: torch.fx.GraphModule):
     # Sometimes patch_functions() misses things already in the graph
```

Comment:
What is AutogradMonkeypatch used for? do i also need to update it?  not obvious what it means that 'replacements' "is" the function being called on torch_function

(see above, L27, not in this diff)

Reply:
That looks like a typo to me.  I think it should be: `func in replacements`.

Reply:
ok makes sense. i'll PR that too but separately in case it breaks something :)